tools/respec2html.js

```diff
@@ -133,18 +131,35 @@ const usageSections = [
     return process.exit(0);
   }
   const src = new URL(parsedArgs.src, `file://${process.cwd()}/`).href;
-  const whenToHalt = {
-    haltOnError: parsedArgs.haltonerror,
-    haltOnWarn: parsedArgs.haltonwarn,
-  };
   const out = parsedArgs.out;
+
   try {
-    await fetchAndWrite(src, out, whenToHalt, {
+    const { html, errors, warnings } = await convertToHTML(src, {
       timeout: parsedArgs.timeout * 1000,
+      onError(error) {
+        console.error(
+          colors.error(`💥 ReSpec error: ${colors.debug(error.message)}`)
+        );
+      },
+      onWarning(warning) {
+        console.warn(
+          colors.warn(`⚠️ ReSpec warning: ${colors.debug(warning.message)}`)
+        );
+      },
```

Comment:
Keeping these as is, as they'll be changed when https://github.com/w3c/respec/pull/3062 comes in anyway